client/guidestours/style.scss

```diff
@@ -85,46 +129,38 @@ $zoom-scale: 5; // the multiplier determining the size of the animated rings
 	animation-delay: #{ $animation-speed / 4 };
 }
 
-.guidestours__choice-button-row {
-	button {
-		width: 48%;
-	}
-	button:nth-child(1) {
-		margin-right: 4%;
-	}
-	button.guidestours__secondary-button {
-		background: $gray-light;
-		color: darken( $gray, 20% );
+// the overlay with animation present when showing a bullseye
+@keyframes guidestours__overlay-animation {
+	0%, 100% {
+		opacity: 0.4;
 	}
-}
 
-.guidestours__single-button-row {
-	button {
-		width: 100%;
+	50% {
+		opacity: 0.7
 	}
 }
 
-.guidestours__external-link,
-.guidestours__bullseye-instructions {
-	p {
-		color: darken( $gray, 10 );
-		margin-bottom: 0;
-		font-style: italic;
-	}
-
-	.gridicon {
-		position: relative;
-		top: 7px;
-	}
-
-	.external-link {
-		border-top: 1px solid $gray-light;
-		display: block;
-		padding-top: 8px;
-		margin-top: 16px;
-	}
+.guidestours__overlay,
+.guidestours__overlay:hover,
+.guidestours__overlay:focus,
+.guidestours__overlay:visited,
+.guidestours__overlay:active {
+	box-shadow: inset 0 0 0 1px rgba( $gray-light, 0.4 ),
+			0 0 12px 1px rgba( $gray-dark, 0.4 ),
+			0 0 0 9999px rgba( $gray-light, 0.8 );
+	transition: box-shadow 300ms ease-in-out;
+	z-index: z-index( 'root', '.web-preview' );
```

Comment:
I suspect we'll want this to stack over the `.fullscreen-fader`, or perhaps we should add a new layer config for the overlay here: https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/functions/_z-index.scss#L133